test/e2e/feature/feature.go

```diff
@@ -363,6 +363,10 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	ValidatingAdmissionPolicy = framework.WithFeature(framework.ValidFeatures.Add("ValidatingAdmissionPolicy"))
 
+	// Owned by SIG Storage
+	// Tests related to VolumeAttributesClass (https://kep.k8s.io/3751)
+	VolumeAttributesClass = framework.WithFeature(framework.ValidFeatures.Add("VolumeAttributesClass"))
```

Comment:
Please add a to-do item, for example,

// TODO: This label only requires the storage.k8s.io/v1alpha1 and its feature-gate enabled. We can remove it once https://github.com/kubernetes/kubernetes/pull/124350 is merged.

Reply:
Added TODO comment